lib/dependabot/file_updaters/java_script/base.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require "dependabot/file_updaters/base"
+require "dependabot/shared_helpers"
+
+module Dependabot
+  module FileUpdaters
+    module JavaScript
+      class Base < Dependabot::FileUpdaters::Base
+        def updated_dependency_files
+          lockfile_name = self.class::LOCKFILE_NAME
+          [
+            updated_file(
+              file: package_json,
+              content: updated_dependency_files_content["package.json"]
+            ),
+            updated_file(
+              file: lockfile,
+              content: updated_dependency_files_content[lockfile_name]
```

Comment:
Could do `lockfile.name` here and remove the `lockfile_name` variable on line 11.